src/java/org/apache/nutch/parse/ParseUtil.java

```diff
@@ -237,7 +237,7 @@ public void processSitemapParse(String url, WebPage page,
           try {
             reversedUrl = TableUtil.reverseUrl(toUrl); // collect it
           } catch (MalformedURLException e) {
-            e.printStackTrace();
+            LOG.error("MalformedURLException occurred: ", e);
```

Comment:
Hi @kamaci, thanks. That's of course better than the current state. But at least some messages could be more informative.